anvil/catalog/ftd_consensus/models/GWAS/unharmonized/GWAS_ftd_accesscontrolledrecord.sql

```diff
@@ -0,0 +1,14 @@
+{{ config(materialized='table', schema='GWAS_data') }}
+
+select 
+  {{ generate_global_id(prefix='',descriptor=[''], study_id='GWAS') }}::text as "has_access_policy",
+    {{ generate_global_id(prefix='',descriptor=[''], study_id='GWAS') }}::text as "id"
+from {{ ref('GWAS_stg_bmi') }} as bmi
+join {{ ref('GWAS_stg_demographics') }} as demographics
+on subjectconsent.subject_id = demographics.subject_id  join {{ ref('GWAS_stg_pedigree') }} as pedigree
+on   join {{ ref('GWAS_stg_phecode') }} as phecode
+on   join {{ ref('GWAS_stg_sampleattributes') }} as sampleattributes
+on samplesubjectmapping.sample_id = sampleattributes.sample_id  join {{ ref('GWAS_stg_samplesubjectmapping') }} as samplesubjectmapping
+on sampleattributes.sample_id = samplesubjectmapping.sample_id  join {{ ref('GWAS_stg_subjectconsent') }} as subjectconsent
+on demographics.subject_id = subjectconsent.subject_id 
+
```

Comment:
For unharmonized tables it is best practice to clean it up before merging so the docs aren't confusing.  Especially if you are running all of the models in the end. The tables are generated this way to keep dbt from having compiling issues but should not be left this way when the tgt model is run.

Change to something like this for each of them. Setting their values to NULL.
select 
    NULL::text as "has_access_policy",
   NULL::text as "id"